pkg/cdi/container-edits.go

```diff
@@ -280,6 +295,20 @@ func (m *Mount) Validate() error {
 	return nil
 }
 
+// IntelRdt a wrapper used for validating IntelRdt configuration.
+type IntelRdt struct {
+	*specs.IntelRdt
+}
+
+// ValidateIntelRdt validates the IntelRdt configuration.
+func ValidateIntelRdt(i *specs.IntelRdt) error {
+	// ClosID must be a valid Linux filename
+	if len(i.ClosID) >= 4096 || i.ClosID == "." || i.ClosID == ".." || strings.ContainsAny(i.ClosID, "/\n") {
```

Comment:
> Is this supposed to check for a path in the current folder?

No, it will practically be a subdir createated under `/sys/fs/resctrl/`. 

> Are there `filepath` functions that can do this for us to make this easier to read?

No, I don't think so. Linux does not have much restrictions on filenames (I think newline is a special case for the `resctrlfs` pseudo-filesystem as is the length). Another possibility is just forget about input validation here and let the low-level runtime engine do it.